src/_scss/pages/awardV2/visualizations/overview/overview.scss

```diff
@@ -22,6 +24,9 @@
             }
           }
         }
+        .award-overview-column__spacing {
+          padding: 0 rem(15);
```

Comment:
This needs to be 

.award-overview-column__spacing {
     @include media($tablet-screen) {
          padding: 0 rem(15);
     }
}